bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/internal/XbmcActiveBinding.java

```diff
@@ -326,6 +328,8 @@ protected void internalReceiveCommand(String itemName, Command command) {
                 connector.playerOpenPVR(command.toString(), 2);
             } else if (property.equals("PVR.OpenRadio")) {
                 connector.playerOpenPVR(command.toString(), 1);
+            } else if (property.equals("Refresh")) {
+                execute();
```

Comment:
Are you certain that it is threadsafe to do this?  The `execute()` method is called at some refreshInterval in another thread, so it may be possible that this method could be run from each thread overlapping.  You might consider using a technique like in the HTTP or Ecobee binding, where the execute() method is called based on a short `granularity` (five seconds, for example), but checks a future timestamp to see if it should do anything.  The "Refresh" action would then just expire the future timestamp, so the next scheduled `execute` happens on average `granulary/2` into the future.  The `execute()` method would just set the future timestamp to the current timestamp + refreshInterval.  (There are certainly other sound approaches to avoiding thread safety issues.)